02_deploying_to_kubernetes.md

```diff
@@ -550,7 +568,10 @@ In this case, the type is `LoadBalancer`, which makes the exposed Pods accessibl
 
 The default Service type is `ClusterIP`, which makes the exposed Pods only accessible from within the cluster.
 
-> **Pro tip:** find out about all available Service types with `kubectl explain service.spec.type`.
+> **Pro tip:** find out about all available Service types with
+> ```
+> kubectl explain service.spec.type
+> ```
```

Comment:
This is a limitation of our markdown engine, but the blockquote will break with the current change as the triple ticks are rendered as a full editor.